src/routes/SlideMenu.svelte

```diff
@@ -0,0 +1,73 @@
+<script>
+	export let open = false;
+	export let duration = 0.15;
+</script>
+
+<div class="relative w-full">
+	<label class="menu">
+		<input type="checkbox" class="menu hidden" bind:checked={open} />
+		<svg
+			style="--duration:{duration}"
+			xmlns="http://www.w3.org/2000/svg"
+			width="24"
+			height="24"
+			viewBox="-12 -12 24 24"
+			fill="none"
+			stroke="currentColor"
+			stroke-width="2"
+			stroke-linecap="round"
+			stroke-linejoin="round"
+			class="feather feather-menu"
+			><line class="line1" x1="-9" y1="0" x2="9" y2="0" />
+			<line class="line2" x1="-9" y1="0" x2="9" y2="0" /><line
+				class="line3"
+				x1="-9"
+				y1="0"
+				x2="9"
+				y2="0"
+			/>
+		</svg>
+	</label>
```

Comment:
I made it so the menu button stays in the same place @goingawil

Reply:
Cool! When I click the menu button now, it sometimes highlights the "A menu item" text, but that's not a big deal. I won't resolve this conversation yet in case there's something you wanna do about that, but otherwise you can resolve the convo.